src/mixins/filing-template-mixin.ts

```diff
@@ -136,9 +136,6 @@ export default class FilingTemplateMixin extends DateMixin {
       }
     }
 
-    // delete invalid tax id (see ticket 15122)
-    delete filing.business.taxId
-
```

Comment:
Now that the schema has been updated, the Tax Id (aka Business Number) can be saved in the business objects, even if they are BN9 or BN15.

Ref: https://github.com/bcgov/business-schemas/blob/f70aef8ff39ac6ef2cd6194362aa10ac14260197/src/registry_schemas/schemas/business.json#L159